src/ui/public/courier/fetch/request/_error_handler.js

```diff
@@ -14,7 +14,7 @@ define(function (require) {
       });
 
       if (!myHandlers.length) {
-        notify.fatal(new Error('unhandled error ' + (error.stack || error.message)));
+        notify.fatal(new Error(`unhandled courier request error: ${ notify.describeError(error) }`));
```

Comment:
Technically this changes the behavior of this particular handler, since the previous behavior was to use `error.stack` if it did exist, whereas `describeError` will use `error.message` regardless.  That's probably not a big deal though since this only used in fatal exceptions, so nothing would programmatically be relying on the contents of the string.